site/styles/main.scss

```diff
@@ -349,3 +349,24 @@ s72-classification-label {
 h1, h2, h3, h4, h5, h6, p, .title {
   word-wrap: break-word;
 }
+
+p, .s72-prompt, .s72-searchresults-empty-content, span, .s72-modal-body .s72-shopping-done {
+a {
+  color: $body-color;
+  text-decoration: underline;
+  background-color: transparent;
+  font-weight: bold;
+  }
+  a:hover, a:focus {
+    color: $secondary;
+    text-decoration: none; }
+
+a:not([href]):not([tabindex]) {
```

Comment:
Not sure I understand what all this `not` stuff is for... carousel left/right buttons?